semantic_text_search/.ipynb_checkpoints/semantic_text-checkpoint.ipynb

```diff
@@ -44,7 +44,7 @@
   },
```

Comment:
<p>We should remove this entire jupyter checkpoints folder from the repo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>